spotify_schema.json

```diff
@@ -0,0 +1,66 @@
+{
+    "$schema": "http://json-schema.org",
+    "$id": "http://yio-remote.com/spotify.json",
+    "type": "object",
+    "title": "YIO Integration Spotify Schema",
+    "description": "Required data points to set up a Spotifyintegrations.",
+    "default": {},
+    "additionalProperties": true,
+    "examples": [
+        {
+            "access_token": "",
+            "client_id": "",
+            "client_secret": "",
+            "entity_id" :""
+        }
+    ],
+    "required": [
+        "access_token",
+        "client_id",
+        "client_secret",
+        "entity_id"
+    ],
+    "properties": {
+        "access_token": {
+            "$id": "#/properties/access_token",
+            "type": "string",
+            "title": "The access_token schema",
+            "description": "Not user input. Access token generated after authenticating to Spotify.",
+            "default": "",
+            "examples": [
+                "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJ1c2VyIjoiVlx1MDAxNcKbwoNUwoonbFPCu8KhwrYiLCJpYXQiOjE0NDQyNjI1NDMsImV4cCI6MTQ0NDI2MjU2M30.MldruS1PvZaRZIJR4legQaauQ3_DYKxxP2rFnD37Ip4"
```

Comment:
Just to make sure: is this a dummy token? Not that we publish a real access token...
Same applies to client_id and client_secret

Reply:
All these are dummy :)